src/main/java/org/jetbrains/bsp/bazel/server/bazel/BazelRunnerBuilder.java

```diff
@@ -79,11 +79,11 @@ public BazelRunnerBuilder withKinds(List<BazelQueryKindParameters> bazelParamete
     return this;
   }
 
-  public BazelProcessResult executeBazelCommand() {
+  public BazelProcess executeBazelCommand() {
```

Comment:
well, this method is used at the very beginning of the server, even before we know the bes server port

Reply:
But I'm not sure there's any real reason as to why that should be done in that order

Reply:
well, it's the consequence of this circular initialization, nevertheless i'll try to change it

Reply:
after little research, it will make entire proces even more ambiguous, bo maybe I'll leave it as it is now. I think that there is nothing bad in those 2 separate methods